src/data/data.tsx

```diff
@@ -1,15 +1,15 @@
 export const navLinks = [
-  { name: "00 Home", path: "/" },
+  { name: "00 HOME", path: "/" },
   {
-    name: "01 Destination",
+    name: "01 DESTINATION",
     path: "/destination",
   },
   {
-    name: "02 Crew",
+    name: "02 CREW",
     path: "/crew",
   },
   {
-    name:  "03 Technology",
+    name:  "03 TECHNOLOGY",
```

Comment:
Why not leave the text capitalized and change its look with `text-transform`?

Reply:
it's a good way too, i just avoided adding more stylization

Reply:
Any specific reason why you wouldn’t want to add more stylization? The reason I ask is bc I want to know the thought process that goes into building/styling with CSS-in-JS or `.tsx` files 😁 It’s ok if it’s just like “ah, it’s a personal project, I didn’t wanna lose time” or something